components/sync_manager/android/src/main/java/mozilla/appservices/syncmanager/RustError.kt

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2018 Mozilla
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Hmm, is this Apache because it's FFI, or? 🤔

Reply:
Nope, looks like the file I copied it from weirdly is sill apache2...